ftplugin/latex-suite/main.vim

```diff
@@ -412,14 +412,15 @@ function! Tex_GetMainFileName(...)
 		exe 'cd '.dirLast
 	endwhile
 
-	let lheadfile = glob('*.latexmain')
+	let lheadfile = glob('*.latexmain',v:true)
 	if lheadfile != ''
 		" Remove the trailing .latexmain part of the filename... We never want
 		" that.
 		let lheadfile = fnamemodify(substitute(lheadfile, '\.latexmain$', '', ''), modifier)
 	else
 		" If we cannot find any main file, just modify the filename of the
 		" current buffer.
+		echo "XXX No found this: ".lheadfile)
```

Comment:
Please remove this debug statement (or use `Tex_Debug`).

Reply:
Sorry. I've put up another pull request with this cleaned up.